src/test-wast-parser.cc

```diff
@@ -86,3 +87,33 @@ TEST(WastParser, LongTokenSpace) {
   ASSERT_STREQ(R"(unexpected token notmodule, expected EOF.)",
                errors[1].message.c_str());
 }
+
+TEST(WastParser, InvalidBinaryModule) {
+  std::string text = R"((module binary "\00asm\bc\0a\00\00"))";
+  std::vector<uint8_t> expected_data = {
+      0, 'a', 's', 'm', static_cast<uint8_t>('\xbc'), '\x0a', 0, 0};
```

Comment:
This static_cast seems a little odd.  Is there a better way to write this?   Is this needed because C chars are signed by default?    Does 0xbc work here instead?